### ISSUE TITLE
Question about TrivialQA hyperparameter

### ISSUE THREAD

**Issue by Fu-Dayuan** [solidsea98/Neural-Corpus-Indexer-NCI#2]
How many epochs did you train?  
Is there any epoch-loss graph?
And could you tell me the time required for training in your setting？ (a day or a week or something else)
I try to use 2  RTX 3090 to train this model and find that it may take me more than 40 days (in 500 epochs, which is the default setting)

**Comment by Fu-Dayuan** [comment]
and in "main_model.py" , class "l1_query", function " __getitem__",  its return has 2 variables that have not been set.
![image](https://user-images.githubusercontent.com/77482343/218504345-f8fff817-5ca6-4805-aeeb-c6286e4f9b87.png)

**Comment by solidsea98** [comment]
Thank you for your attention. It takes about 50 epochs to fully converge, and it takes about a week to use 8 NVIDIA Tesla V100 32GB. Epoch-loss related diagrams are as follows,
![image](https://user-images.githubusercontent.com/31641542/218674049-e1326e03-deb5-4886-92df-fed9b71bea8b.png)
